default/stringtables/es.txt

```diff
@@ -1173,29 +1173,29 @@ Escalado de coste de partes
 #*RULE_STARLANES_EVERYWHERE_DESC
 #* <not translated 1970-01-01 01:00:00>
 
-#*RULE_HABITABLE_SIZE_TINY
-#* <not translated 1970-01-01 01:00:00>
+RULE_HABITABLE_SIZE_TINY
+Hábitats en planetas enanos
 
-#*RULE_HABITABLE_SIZE_SMALL
-#* <not translated 1970-01-01 01:00:00>
+RULE_HABITABLE_SIZE_SMALL
+Hábitats en planetas pequeños
 
-#*RULE_HABITABLE_SIZE_MEDIUM
-#* <not translated 1970-01-01 01:00:00>
+RULE_HABITABLE_SIZE_MEDIUM
+Hábitats en planetas medianos
 
-#*RULE_HABITABLE_SIZE_LARGE
-#* <not translated 1970-01-01 01:00:00>
+RULE_HABITABLE_SIZE_LARGE
+Hábitats en planetas  grandes
```

Comment:
De acuerdo, mirando el fichero, he visto que es común poner dos espacios después de un . o un ?, ¿es esto otro error o tiene algún sentido dentro del código? Por meter también esos cambios si hace falta

Reply:
I think there should be no double spaces. If there are double spaces (in the English or the Spanish stringtables) after each period or interrogation, I don't know if that is because of some FOCS requirement (I don't think so) or for some aesthetical reason. I would remove them all but I'm unsure. @geoffthemedio ?

Reply:
I'd remove all double spaces that aren't there for specific reasons. Different people wrote bits of stringtable over many years, so there are probably some cases where they are present, though.